sjb/config/test_cases/push_origin_metrics_release.yml

```diff
@@ -0,0 +1,66 @@
+---
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+timer: 'H H * * *'
+parameters:
+  - name: buildId
+    description: "Unique build number for each run."
+  - name: REPO_OWNER
+    description: "GitHub org that triggered the job."
+    default_value: openshift
+  - name: REPO_NAME
+    description: "GitHub repo that triggered the job."
+    default_value: origin-metrics
+  - name: buildId
+    description: "Unique build number for each run."
+  - name: REPO_OWNER
+    description: "GitHub org that triggered the job."
+    default_value: openshift
+  - name: REPO_NAME
+    description: "GitHub repo that triggered the job."
+    default_value: origin
+  - name: PULL_BASE_REF
+    description: "Ref name of the base branch."
+  - name: PULL_BASE_SHA
+    description: "Git SHA of the base branch."
+  - name: PULL_REFS
+    description: "All refs to test."
+  - name: PULL_NUMBER
+    description: "Pull request number."
+  - name: PULL_PULL_SHA
+    description: "Pull request head SHA."
+sync_repos:
+  - name: "origin-metrics"
+actions:
+  - type: "script"
+    title: "build origin-metrics release"
+    repository: "origin-metrics"
+    script: |-
+      git_branch=$( git rev-parse --abbrev-ref HEAD )
+      echo "Building origin-metrics images on git branch $git_branch"
+      hack/build-images.sh
+  - type: "script"
+    title: "create directory for docker config"
+    script: |-
+      mkdir -p /tmp/.docker
+      chmod a+rwx /tmp/.docker
+  - type: "host_script"
+    title: "transfer docker config to remote host"
+    script: |-
+      scp -F ./.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.docker/config.json openshiftdevel:/tmp/.docker/
+  - type: "script"
+    title: "push origin-metrics release"
+    repository: "origin-metrics"
+    script: |-
+      sudo chmod a+rw /tmp/.docker/config.json
```

Comment:
Please keep the tagging login from the logging job.  If push release doesn't support multi tagging, please add that to your hack/push-release.

Reply:
Oops, ignore this.  I haven't added it yet.  As long as this consistent with logging_release you're good for now.